apps/lead/tasks.py

```diff
@@ -43,8 +43,9 @@ def generate_previews(lead_ids=None):
         extract_from_lead.apply_async((lead_id,), countdown=1)
 
 
+@shared_task(bind=True, max_retries=10)
```

Comment:
For this, we don't need to do retries as we are running this in schedule.
Let's just use @shared_task